interfaces/automated-testing/scd/scd.yaml

```diff
@@ -57,4 +76,26 @@ paths:
         '404':
           description: The USS automated testing interface is not activated.
       summary: Retrieve the status of the USS automated testing interface
-      description: Get the status of the USS automated testing interface
\ No newline at end of file
+      description: Get the status of the USS automated testing interface
+
+  /inject/flight:
+    summary: Process operator details
+    put:      
```

Comment:
This isn't a reasonable endpoint without a request body, so let's at least put a TODO comment ([where the `requestBody` will go](https://github.com/interuss/dss/blob/744b7a4733c785066802533b9aa80485fc68db62/interfaces/automated-testing/rid/injection.yaml#L23) eventually) indicating that the request body will be added later.

Reply:
Thanks! I have now added the todo.